__tests__/app.test.js

```diff
@@ -324,7 +324,37 @@ describe("GET /api/articles/:article_id", () => {
     });
   });
 
-
+  describe("GET /api/users", () => {
+    test("200: Responds with an array of user objects, each having username, name, and avatar_url", () => {
+      return request(app)
+        .get("/api/users")
+        .expect(200)
+        .then(({ body }) => {
+          expect(body.users).toBeInstanceOf(Array);
```

Comment:
This assertion becomes redundant as we use a forEach in this test - if it wasn't an array we wouldn't be able to and the test would fail so we don't explicitly need to check that the articles are an array :slightly_smiling_face: